server/src/main/scala/com/cloudera/livy/server/interactive/InteractiveSession.scala

```diff
@@ -121,13 +121,18 @@ class InteractiveSession(
     }
     builderProperties.put(RSCConf.Entry.SESSION_KIND.key, kind.toString)
 
-    mergeConfList(livyJars(livyConf), LivyConf.SPARK_JARS)
+    require(livyConf.get(LivyConf.LIVY_SPARK_VERSION) != null)
+    require(livyConf.get(LivyConf.LIVY_SPARK_SCALA_VERSION) != null)
+
+    val sparkVersion = LivySparkUtils.formatSparkVersion(livyConf.get(LivyConf.LIVY_SPARK_VERSION))
```

Comment:
how about changing it to 
```
val (sparkMajorVersion, sparkMinorVersion) = ...
```
Otherwise the following 2 statements would make me think they return the same type. 
```
val sparkVersion = LivySparkUtils.formatSparkVersion(livyConf.get(LivyConf.LIVY_SPARK_VERSION))
val scalaVersion = livyConf.get(LivyConf.LIVY_SPARK_SCALA_VERSION)
```